lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +19,23 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (isinstance(text, str) and text):
+        return None
+    punctuation = r'''!"“”#$%&'()*+,-–—./:;<=>?@[\\]^_`{|}~¡¿⟨⟩«»'…⋯‹›'''
+    txt = ''
+    list_with_phrases = []
+    for char in text.replace('\n', ' '):
+        if char in punctuation:
+            if txt:
+                no_blank_lines = txt.strip()
+                if no_blank_lines:
+                    list_with_phrases.append(no_blank_lines)
+                txt = ''
+            continue
+        txt += char
+    if txt:
```

Comment:
why did you check that txt includes something except spaces in the previous iterations but not here?